compiler/build.gradle

```diff
@@ -54,47 +54,68 @@ def addLibraryIfNotLinked = { libName, argList ->
 }
 
 model {
-  toolChains {
-    gcc(Gcc) {
-      target("x86_64") {
-        cppCompiler.withArguments { args ->
-          args << "-m64"
-        }
-        linker.withArguments { args ->
-          args << "-m64"
-        }
+  if (System.getProperty('use.vc') != null) {
```

Comment:
I don't like this code (no offence). I think that there should be no switching in so weird way. 

The easiest way to achieve behaviour that we want is add ```visualCpp``` toolchain and in platforms put this code: ```architecture osdetector.os == 'windows' ? 'x86' : 'local_arch'```

I've got prepared branch in my work, but I'm w8 for approval to send it out (privacy policy).